test/katex-spec.js

```diff
@@ -811,15 +811,15 @@ describe("A text parser", function() {
 
 describe("A texvc builder", function() {
     it("should not fail", function() {
-        expect("\\lang\\N\\darr\\R\\dArr\\Z\\Darr\\alef\\rang").toBuild();
-        expect("\\alefsym\\uarr\\Alpha\\uArr\\Beta\\Uarr\\Chi").toBuild();
-        expect("\\clubs\\diamonds\\hearts\\spades\\cnums\\Complex").toBuild();
-        expect("\\Dagger\\empty\\harr\\Epsilon\\hArr\\Eta\\Harr\\exist").toBuild();
-        expect("\\image\\larr\\infin\\lArr\\Iota\\Larr\\isin\\Kappa").toBuild();
-        expect("\\Mu\\lrarr\\natnums\\lrArr\\Nu\\Lrarr\\Omicron").toBuild();
-        expect("\\real\\rarr\\plusmn\\rArr\\reals\\Rarr\\Reals\\Rho").toBuild();
-        expect("\\text{\\sect}\\sdot\\sub\\sube\\supe").toBuild();
-        expect("\\Tau\\thetasym\\weierp\\Zeta").toBuild();
+        expect`\lang\N\darr\R\dArr\Z\Darr\alef\rang`.toBuild();
+        expect`\alefsym\uarr\Alpha\uArr\Beta\Uarr\Chi`.toBuild();
```

Comment:
I thought `\u` didn't work in raw strings because it's a Unicode escape. See e.g.

https://github.com/KaTeX/KaTeX/blob/bedce93260cc413abb7730a5693dc27222297178/test/katex-spec.js#L2919

Is that incorrect? (It might be; I'm not sure.)

Reply:
No, you right. I'm changing it to double backslash 
![image](https://github.com/KaTeX/KaTeX/assets/8845364/9144412b-198b-4123-adb5-8f088c84bd8e)

Reply:
This isn't the right experiment though. I think ``String.raw`...` `` is closer, but I couldn't immediately find how we're providing template strings creation functions.